src/router/index.ts

```
@@ -8,9 +9,22 @@ const router = createRouter({
       return savedPosition
     } else {
-      return { top: 0, left: 0, behavior: 'smooth' };
+      return { top: 0, left: 0, behavior: 'smooth' }
     }
+  },
+})
 
+// reroute when address is undefined
+router.beforeEach((to, from, next) => {
+  const store = useWalletStore()
+  if (to.meta && to.meta.noAuth) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)